lib/auth/apiserver.go

```diff
@@ -97,6 +97,9 @@ func NewAPIServer(config *APIConfig) http.Handler {
 	srv.POST("/:version/users/:user/ssh/authenticate", srv.withAuth(srv.authenticateSSHUser))
 	srv.GET("/:version/users/:user/web/sessions/:sid", srv.withAuth(srv.getWebSession))
 	srv.DELETE("/:version/users/:user/web/sessions/:sid", srv.withAuth(srv.deleteWebSession))
+
+	// User signup tokens.
+	srv.GET("/:version/signuptokens", srv.withAuth(srv.getSignupTokens))
 	srv.GET("/:version/signuptokens/:token", srv.withAuth(srv.getSignupTokenData))
```

Comment:
In Gravity this handler is defined via '/tokens/user/:id` since both Signup and Reset tokens use the same Web API. The client makes the same request to pull the token info for Signup and Reset flows.